extra/resources/controld

```diff
@@ -228,8 +228,12 @@ controld_monitor() {
     killall -0 ${OCF_RESKEY_daemon} >/dev/null 2>&1 ; rc=$?
 
     case $rc in
-      0) dlm_tool ls | grep -q "wait fencing"
-         if [ $? -eq 0 ]; then
+      0) smw=$(dlm_tool status -v | grep "stateful_merge_wait=" | cut -d= -f2)
+         if [ $smw -eq 1 ]; then
```

Comment:
There's error "[: -eq: unary operator expected" if stateful_merge_wait is not available,